src/routes/login.rs

```diff
@@ -0,0 +1,32 @@
+use rocket::{response::Redirect, request::Form, State};
+use rocket_contrib::{database::DbConn, templates::Template};
+use rocket_jwt::{Claims, JWT, Signer, Token};
+use crate::auth::auth;
+
+#[derive(FromForm)]
+struct LoginForm {
+    ip_address: String,
```

Comment:
no need to send ip address in login form

Reply:
log in form needs ip_address

Reply:
this is how an unknowing client connects to server

Reply:
do not hash the ip address for jwt auth

Reply:
Sure let's figure out what we can use instead.

According to https://auth0.com/docs/secure/tokens/json-web-tokens/json-web-token-claims we could use for ex.

`Add a user's email address to an access token and use that to uniquely identify the user.`

Maybe we could shift to having a username/password model for the server I think this would make authentication easier for us.